helper-cli/src/main/kotlin/commands/CreateAnalyzerResultCommand.kt

```diff
@@ -87,9 +87,14 @@ internal class CreateAnalyzerResultCommand : CliktCommand(
                 "-P ort.scanner.storages.postgres.schema=testSchema"
     ).associate()
 
+    private val scanCodeVersion by option(
+        "--scan-code-version",
+        help = "The scan code version to match for equality against the scan results."
```

Comment:
s/scan code/ScanCode/

Reply:
Also "to match for equality against" sounds a bit clumsy... how about just "to match for in the scan results"?